60-database.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# 50-config.py or equivalent must be run first to set omero.db.*
+
+set -eu
+
+omero=/opt/omero/server/OMERO.server/bin/omero
+omego=/opt/omero/omego/bin/omego
+cd /opt/omero/server
+
+CONFIG_omero_db_host=${CONFIG_omero_db_host:-}
+if [ -n "$CONFIG_omero_db_host" ]; then
+    DBHOST="$CONFIG_omero_db_host"
+else
+    DBHOST=db
+    $omero config set omero.db.host "$DBHOST"
```

Comment:
No, `CONFIG_omero_db_host` will already be converted into omero.db.host by `50-config.py`- I'll update the comment at the top of the file. The only reason for special handling here is that the default of `omero.db.host=localhost` will never work, instead it defaults to `db` which should be set by `docker run --link ...`.